ngraph/core/src/op/matmul.cpp

```diff
@@ -113,68 +53,138 @@ shared_ptr<Node> op::MatMul::clone_with_new_inputs(const OutputVector& new_args)
 
 namespace
 {
-    Shape evaluate_matmul_output_shape(const Shape& arg0_shape,
-                                       const Shape& arg1_shape,
-                                       bool transpose_a,
-                                       bool transpose_b)
+    PartialShape evaluate_matmul_output_shape(const PartialShape& arg0_shape,
```

Comment:
I am not sure if it is the best name. evaluate prefix suggest calling ref implementation

Reply:
You are right, function name is changed.